src/main/res/values/strings.xml

```diff
@@ -324,5 +326,6 @@
     <string name="size_largest">Size (largest first)</string>
     <string name="size_smallest">Size (smallest first)</string>
     <string name="file_type">File Type</string>
+    <string name="preference_key_language" translatable="false">bn-rIN</string>
```

Comment:
This looks suspicious. Seems incorrect. @chirag-jn ?

Reply:
That key works with any value since the value acts like a placeholder only. However, you are right, I missed that, the correct value should have been the one which was present already. I'll update it.